src/textual/app.py

```diff
@@ -979,6 +1001,28 @@ async def run_app() -> None:
             event_loop.run_until_complete(run_app())
         return self.return_value
 
+    @contextmanager
+    def suspend(self) -> Iterator[None]:
```

Comment:
Very minor thing this, but I wonder if this would be better called `suspended` rather than `suspend`:

```py
with self.suspended():
   ...
```

sort of reads a little better, ish, kinda? Maybe?

@JoshKarpel, @willmcgugan, any preferences?

Reply:
suspend fits a convention of methods starting with verbs. Suspended reads like a property to me.

Reply:
https://peps.python.org/pep-0343/#examples suggests the names suspended and paused over suspend and pause since these are contextmanagers that take an action in enter and undo it during exit. (This is consistent w/ my intuition / preference too.)